docs/configuration.md

```diff
@@ -76,6 +76,65 @@ systems.
 Section for Worker configurations that should only be applied to Windows-based
 systems.
 
+### status
+
```

Comment:
Missing the "intro" paragraph from before, trying to describe what the "status" actually does...

```
Watchmaker supports posting the watchmaker status to status providers. Watchmaker
status values are one of: 'Running', 'Failed', or 'Completed'. Each status provider
defines what it means to "post the status". Currently, the supported provider
types include: 'aws' and 'azure'. These status providers both post the status as
a tag to the instance/VM.
```

Reply:
Fixed